doc/nrf/ug_nrf9160.rst

```diff
@@ -162,6 +162,31 @@ If the modem is shut down gracefully before the next boot (by using ``AT+CFUN=0`
 
 For more detailed information, see the `system mode section in the AT Commands reference document`_.
 
+.. |An nRF9160-based device| replace:: An nRF9160 DK
+.. |an nRF9160-based device| replace:: an nRF9160 DK
+
+.. nrf9160_gps_lte_start
+
+Concurrent GPS and LTE
+======================
+
+|An nRF9160-based device| supports GPS in LTE-M and NB-IoT.
+Concurrent operation of GPS with optional power saving features such as extended Discontinuous Reception (eDRX) and Power Saving Mode (PSM) is also supported and recommended.
```

Comment:
Add comma:
Concurrent operation of GPS with optional power saving features, such as extended Discontinuous Reception (eDRX) and Power Saving Mode (PSM), is also supported and recommended.